packages/shared/src/components/modals/SharePostModal.tsx

```diff
@@ -34,14 +37,15 @@ export default function PostModal({
     offset: showArticleOnboarding ? ONBOARDING_OFFSET : 0,
   });
   const containerClass = 'post-content';
-  const isPublicSquad = isSourcePublicSquad(post?.source);
+  const isPublicSquad = isSourcePublicSquad(postSource);
```

Comment:
using `post.source` causes jittering UI since the modal size depends on this. `postSource` is present in props before the `post` record is loaded for this reason